core/services/functions/connector_handler.go

```diff
@@ -83,6 +83,7 @@ func NewFunctionsConnectorHandler(pluginConfig *config.PluginConfig, signerKey *
 		allowedHeartbeatInitiators[strings.ToLower(initiator)] = struct{}{}
 	}
 	return &functionsConnectorHandler{
+		// pluginConfig.GatewayConnectorConfig.NodeAddress undefined (type *invalid type has no field or method NodeAddress)
```

Comment:
It's looking for what you moved as core/config `ConnectorGatewayConfig`, but as a struct